fhir-persistence-jdbc/src/test/resources/test.jdbc-perform.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Properties for JDBC performance tests in the fhir-persistence-jdbc project
+# Default datastore is a DB2 fhirdata database.
+
+#Derby embedded properties
+#Don't need any configuration for derby embedded
+
+#Derby network properties
+#dbUrl = jdbc:derby://localhost:1527/fhirdb
+
+#DB2 properties
+dbDriverName = com.ibm.db2.jcc.DB2Driver
+db.host=localhost
+db.port=50000
+db.database=fhirdb
+user=db2inst1
+password=change-me
+updateCreateEnabled = true
+schemaName = FHIRDATA
```

Comment:
this is minor, but I think we should have more consistent formatting here.  pick one:
* `prop=value` or
* `prop = value`

Reply:
done